torchgeo/datasets/chesapeake.py

```diff
@@ -28,23 +28,14 @@
 class Chesapeake(RasterDataset, abc.ABC):
     """Abstract base class for all Chesapeake datasets.
 
-    `Chesapeake Bay High-Resolution Land Cover Project
-    <https://www.chesapeakeconservancy.org/conservation-innovation-center/high-resolution-data/land-cover-data-project/>`_
+    `Chesapeake Bay Program Land Use/Land Cover Data Project
+    <https://www.chesapeakeconservancy.org/conservation-innovation-center/high-resolution-data/lulc-data-project-2022/>`_
```

Comment:
@calebrob6 I didn't look into this too much but it seems like there's a new version of this dataset. Not sure if we should update the link or link to the old one instead, I don't know how different they are. Let me know if I should revert this.

Reply:
It looks like the new version has different classes

Reply:
Alright, let me undo this change and we can create a v2 someday if we want.